LernieControl/LernieControl.lua

```diff
@@ -0,0 +1,83 @@
+--[[
+    EnemyControl
+    Authors:
+        SleepSoul (Discord: SleepSoul#6006)
+    Dependencies: ModUtil, RCLib
+    Change the pool of side heads that can be chosen in the Lernie fight.
+]]
+ModUtil.Mod.Register("LernieControl")
+
+local config = {
+    LernieSetting = "Vanilla",
+    AllowIneligibleCombos = false, -- Normally Lernie side heads cannot be the same as the side head, this overrides that. Note that the game can crash if this is not enabled and preset only contains 1 head
+}
+LernieControl.config = config
+
+LernieControl.Presets = {
+    Vanilla = {
+        Heads = {},
+        InheritVanilla = true,
```

Comment:
is there a reason `inheritVanilla` would ever be false? seemed a bit confusing that all configs have it set to true, even the vanilla one itself. If there's not a real case for it then it would lead to simplifying some of the logic a bit

Reply:
could be false if you wanted to make a config that was smaller; for instance, if you just wanted green and orange heads, you could have inheritance off and set those to true rather than setting all the others to false (see RatsOClock in EnemyControl for an example). could probably be left out as a feature here tbf, since the presets will always be small either way

Reply:
Removed in new commit